src/compliance/privacy/magento-privacy-js-library.md

```diff
@@ -0,0 +1,62 @@
+---
+group: compliance
+title: Magento Privacy JavaScript Library
+---
+
+<!-- TODO: Unpublish this topic and create a topic redirect to the adobe privacy js topic when the adobe privacy library is integrated with Magento -->
+
+The Magento Privacy JavaScript Library is a set of tools to help create a process for accessing and deleting private data collected by Magento.
+
+Magento's data tracking services can store private information applicable to privacy regulations such as the [General Data Protection Regulation (GDPR)][] and [California Consumer Privacy Act (CCPA)][].
+This library provides a set of functions for creating privacy-data requests and gathering their responses.
+Use this library to retrieve and remove the data stored in the browser by Magento's data tracking services.
+
+## Installation
+
+Download the Magento Privacy JavaScript Library from the following location: `commerce.adobe.net/magentoprivacy.js`
```

Comment:
Change the language from `downloading` the library from the link to the library being available via cdn at the link.